nautobot_golden_config/utilities/helper.py

```diff
@@ -39,6 +39,9 @@ def get_job_filter(data=None):
     elif data.get("device"):
         query.update({"id": data["device"].values_list("pk", flat=True)})
 
+    if data.get("tag"):
```

Comment:
Should be moved just below `for field in FIELDS:` loop I believe.

Reply:
@itdependsnetworks the fields loop will add based on `f"{field}_id"` but the tag filter is based on slug.

Reply:
Below being the key word, not in the loop

Reply:
Happy to make the change, can you help me understand why changing the order? Building a dictionary of query values shouldn't matter the order it's created.

Reply:
I guess you are correct in that it does not need to be moved, I actually thought it was being overridden when a device. That being said, logically it is 
* Based on non device fields, add filters
* Comment to say "hey, this is different, this is how we handle device fields)
* Device field filter
* Apply GC settings
* Checks for issues

I think it would make more sense to keep like attributes together, even if order does not make a technical difference.

Reply:
Done